src/main/java/our/yurivongella/instagramclone/service/PostService.java

```diff
@@ -0,0 +1,57 @@
+package our.yurivongella.instagramclone.service;
+
+import com.sun.istack.NotNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import our.yurivongella.instagramclone.controller.dto.post.PostCreateRequestDto;
+import our.yurivongella.instagramclone.controller.dto.post.PostReadResponseDto;
+import our.yurivongella.instagramclone.domain.member.Member;
+import our.yurivongella.instagramclone.domain.member.MemberRepository;
+import our.yurivongella.instagramclone.domain.post.PictureURL;
+import our.yurivongella.instagramclone.domain.post.PictureURLRepository;
+import our.yurivongella.instagramclone.domain.post.Post;
+import our.yurivongella.instagramclone.domain.post.PostRepository;
+import our.yurivongella.instagramclone.util.SecurityUtil;
+
+import java.util.List;
+import java.util.Optional;
+
+@RequiredArgsConstructor
+@Service
+public class PostService {
+    final private PostRepository postRepository;
+    final private PictureURLRepository pictureURLRepository;
+    final private MemberRepository memberRepository;
+
+    @Transactional
+    public Long create(PostCreateRequestDto postCreateRequestDto, Long memberId) {
+        Optional<Member> member =memberRepository.findById(SecurityUtil.getCurrentMemberId());
+        Post post = postCreateRequestDto.toPost(member.orElseThrow(() -> new RuntimeException()));
+        post = postRepository.save(post);
```

Comment:
1. `PostController` 에서 `Long id = postService.create(postRequestDto, SecurityUtil.getCurrentMemberId());` 이렇게 호출하고 있는데 `PostService.create` 내부에서 중복해서 `MemberId` 를 호출하고 있어요. 
파라미터로 안받게 수정하면 좋을 것 같습니다.

2. 파라미터로 넘길때 `orElseThrow` 를 넣는건 가독성이 많이 떨어지는 것 같습니다.
`Member` 값을 가져올 때 먼저 처리하면 좋을것 같아요


```suggestion
    public Long create(PostCreateRequestDto postCreateRequestDto) {
        Member member = memberRepository.findById(SecurityUtil.getCurrentMemberId())
                .orElseThrow(() -> new RuntimeException("조회자 정보가 존재하지 않습니다."));

        Post post = postRepository.save(postCreateRequestDto.toPost(member));
```

Reply:
반영하여 수정하였습니다.
https://github.com/INSPJT/instagram_clone_backend/pull/25/commits/1519df54b1d9b36648e15396918ab756c642a84d